src/SignInUser/SignInUser.test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react'; 
+import { shallow } from 'enzyme'; 
+import { SignInUser, mapDispatchToProps } from './SignInUser.js'; 
+import { authenticateUser } from '../Thunks/authenticateUser.js'
+
+describe('SignInUser', () => {
+  let mockUserName; 
+  let mockPassword;
+  let mockEvent
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = shallow(<SignInUser signIn={jest.fn()}/>)
+    mockUserName = 'Rajaa'; 
+    mockPassword = 'password'; 
+    mockEvent = {
```

Comment:
Not sure if this mockEvent is supposed to include this or token: 

mockEvent = {
     target: {
       name: 'token',
       value: 'token12345'
      }
    }